stage/dotnet/dotnet-sdk/README.md

```diff
@@ -65,6 +65,27 @@ foreach (var pullRequest in pullRequests)
 }
 ```
 
+#### Specifying a Base URL
+
+```csharp
+using GitHub;
+using GitHub.Octokit.Client;
+using GitHub.Octokit.Client.Authentication;
+
+var token = Environment.GetEnvironmentVariable("GITHUB_TOKEN") ?? "";
+var request = new RequestAdapter(new TokenAuthProvider(new TokenProvider(token)))
+	.WithBaseUrl("https://api.github.com")
+    .Build();
```

Comment:
This sample should be updated with the latest code from `stage/dotnet/dotnet-sdk/cli/Authentication/PersonalAccessToken.cs`.

Reply:
Yeah I'll need to move the pattern into the other SDKs as well once we decide that this is the way to move forward.